### PR TITLE
Cuttlefish schema

### PR DESCRIPTION
**Pull request by hairyhum**
Schema for https://github.com/rabbitmq/rabbitmq-server/issues/550.

### CODE REVIEW

File: priv/schema/rabbitmq_web_mqtt.schema [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{mapping, "web_mqtt.num_acceptors.tcp", "rabbitmq_web_mqtt.num_tcp_acceptors", 
+    [{datatype, integer}]}.
+{mapping, "web_mqtt.num_acceptors.ssl", "rabbitmq_web_mqtt.num_ssl_acceptors", 
+    [{datatype, integer}]}.
+
+{mapping, "web_mqtt.tcp_config.port", "rabbitmq_web_mqtt.tcp_config.port",
```
Comment (michaelklishin) [inline priv/schema/rabbitmq_web_mqtt.schema]: I think we should remove the "_config" part.

Comment (hairyhum) [inline priv/schema/rabbitmq_web_mqtt.schema]: Do you mean for `tcp_config.port`, or for any `tcp_config` option?

Comment (michaelklishin) [inline priv/schema/rabbitmq_web_mqtt.schema]: All of them.

Comment (hairyhum) [inline priv/schema/rabbitmq_web_mqtt.schema]: And what about `cowboy_opts` and `sockjs_opts`? Should "_opts" part be removed as well?

Comment (michaelklishin) [inline priv/schema/rabbitmq_web_mqtt.schema]: We have "_opts" in other places and they don't seem to be too bad to me. "_config" is something that is clearly extraneous: every single setting in a config file is… well, some kind of configuration ;)

**Pull request merged**